iothub/device/src/Transport/Mqtt/MqttTransportHandler.cs

```diff
@@ -880,6 +885,11 @@ public override async Task SendTwinPatchAsync(TwinCollection reportedProperties,
                             return false; // Let anything else stop the application.
                         });
                     }
+                    catch (DotNetty.Transport.Channels.ConnectException ex)
```

Comment:
(coding std): Please add `using` instead of expanding to FQN.

Reply:
Sure thing